{
	"id": "SUSE-SU-2024:1428-1",
	"modified": "2024-04-24T09:10:46Z",
	"published": "2024-04-24T09:10:46Z",
	"related": [
		"CVE-2024-3651"
	],
	"summary": "Security update for python-idna",
	"details": "This update for python-idna fixes the following issues:\n\n- CVE-2024-3651: Fixed potential DoS via resource consumption via specially crafted inputs to idna.encode() (bsc#1222842).\n",
	"affected": [
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Module for Public Cloud 12",
				"name": "python-idna",
				"purl": "purl:rpm/suse/python-idna&distro=SUSE%20Linux%20Enterprise%20Module%20for%20Public%20Cloud%2012"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.5-3.13.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-idna": "2.5-3.13.1",
						"python3-idna": "2.5-3.13.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Server 12 SP5",
				"name": "python-idna",
				"purl": "purl:rpm/suse/python-idna&distro=SUSE%20Linux%20Enterprise%20Server%2012%20SP5"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.5-3.13.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-idna": "2.5-3.13.1",
						"python3-idna": "2.5-3.13.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "SUSE:Linux Enterprise Server for SAP Applications 12 SP5",
				"name": "python-idna",
				"purl": "purl:rpm/suse/python-idna&distro=SUSE%20Linux%20Enterprise%20Server%20for%20SAP%20Applications%2012%20SP5"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.5-3.13.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python-idna": "2.5-3.13.1",
						"python3-idna": "2.5-3.13.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20241428-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1222842"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2024-3651"
		}
	]
}
